src/components/DeleteAccountModal/DeleteAccountModal.jsx

```diff
@@ -34,12 +54,32 @@ const DeleteAccountModal = () => {
             <Button mr={3} onClick={onClose}>
               Cancel
             </Button>
-            <Button colorScheme="red">Delete Staff Member</Button>
+            <Button colorScheme="red" onClick={onSubmit}>
+              Delete Staff Member
+            </Button>
           </ModalFooter>
         </ModalContent>
       </Modal>
     </>
   );
 };
 
+DeleteAccountModal.propTypes = {
+  staffProfile: PropTypes.arrayOf(
```

Comment:
Get rid of the outer `PropTypes.arrayOf` section because you aren't passing in an array of user data.